packages/web-app-files/src/components/SideBar/Shares/Collaborators/RoleDropdown.vue

```diff
@@ -223,7 +223,8 @@ export default defineComponent({
     window.removeEventListener('keydown', this.cycleRoles)
   },
 
-  mounted() {
+  async mounted() {
+    await this.loadIncomingParentShare.perform(this.resource)
```

Comment:
doesn't this try to load the incoming parent shares for each single `RoleDropdown` component instance? Can we load the incoming parent shares once in the `SideBar.vue` or in the `SharesPanel.vue` and hand it down to the child components via provide/inject (if from SideBar.vue) or via component properties (if coming from SharesPanel.vue)?